openqa-mail-notification/mail_notification.rb

```diff
@@ -89,7 +89,7 @@ def send_notification(to, subject, message)
   new_result = last_build >= frequency_minutes_ago
 
   unless new_result
-    @logger.info("No new builds for #{version}...")
+    @logger.info("No new builds for #{version} right now (#{DateTime.now.new_offset(0)}). Last build: #{last_build}")
```

Comment:
not possible to do `logger.debug` or any other log level other  than info? Info means that we are log it every email. So if you need it for every email, maybe you want to log it into influx instead of flood the log files?

Reply:
Every run has only one log...

![Screenshot from 2021-09-22 13-52-22](https://user-images.githubusercontent.com/514785/134338711-da683a3e-8979-4bb6-9a15-acd936e5e7d7.png)

Reply:
So nothing is flooded